pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -272,7 +272,7 @@ impl InMemoryLayer {
     // Write operations
 
     /// Remember new page version, as a WAL record over previous version
-    pub fn put_wal_record(&self, blknum: u32, rec: WALRecord) -> Result<()> {
+    pub fn put_wal_record(&self, blknum: u32, rec: WALRecord) -> Result<u32> {
```

Comment:
What do these return? Please explain in comment

Reply:
Done